### PR TITLE
[CIVIS-8718] Update documentation for CIVIS GitHub PR comments

### PR DESCRIPTION
**Pull request by ManuelPalenzuelaDD**
We have recently slightly changed the format of the PR comments, so we need to update the example image.

Additionally, in the code coverage docs we were previously not mentioning that we show the coverage change of a PR in the comments, so we've added a mention of that.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by calvarjorge — APPROVED** [review]

File: content/en/tests/code_coverage.md [hunk 1]
```diff
@@ -310,6 +310,10 @@ You can also see the code coverage's evolution on the [Branch Overview page][6]
 {{< img src="/continuous_integration/code_coverage_branch_view.png" text="Branch view's code coverage" style="width:100%" >}}
 
 
+## Show code coverage change of a pull request
+
+The code coverage change of a GitHub pull request compared to the default branch is shown in the pull request's [test summary comment][8].
```
Comment (brett0000FF) [inline content/en/tests/code_coverage.md:315]:
```suggestion
The pull request's [test summary comment][8] shows the code coverage change of a GitHub pull request compared to the default branch.
```
Slight reorder to make active voice.

**Review by brett0000FF — APPROVED** [review]
Looks good! Just one small suggestion.

File: content/en/tests/code_coverage.md (hunk 1 above)
Comment (ManuelPalenzuelaDD) [reply content/en/tests/code_coverage.md]: Thanks for the suggestion!

**Review by KickingTheTV — APPROVED** [review]
LGTM

**Comment by ManuelPalenzuelaDD** [comment]
>Looks good! Just one small suggestion.
>Let me know when you are ready for me to merge this. 👍

Hi @brett0000FF , could you please merge this? Thank you!!

**Pull request merged**